API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
```

Comment:
I can imagine "tags" being useful. People might want to browse for activities with "javascript", "css", or "passwords" for example. 

The Read, Write, Participate framework was used to group and order content on the page (see https://learning.mozilla.org/en-US/activities/web-lit-basics/)—that does not seem necessary. But a tag field could be useful for searching/browsing.